applications/vanilla/controllers/class.postcontroller.php

```diff
@@ -172,17 +172,22 @@ public function discussion($CategoryUrlCode = '') {
 
         touchValue('Type', $this->Data, 'Discussion');
 
-        // See if we should hide the category dropdown.
-        if ($this->ShowCategorySelector) {
+        if (!$UseCategories || $this->ShowCategorySelector) {
```

Comment:
Didn't we just start removing this logic condition everywhere in prep for deprecating that config?